dispatch/dispatch.h

```diff
@@ -49,7 +49,7 @@
 
 #if defined(__linux__) && defined(__has_feature)
 #if __has_feature(modules)
-#include <stdio.h> // for off_t (to match Glibc.modulemap)
+//#include <stdio.h> // for off_t (to match Glibc.modulemap)
```

Comment:
Understood.  We have to solve the duplicate definition of va_list somehow.  See https://github.com/apple/swift-corelibs-foundation/pull/399/files for more context.  The same is required for Foundation.